x-pack/plugins/secops/public/pages/hosts/host_details.tsx

```diff
@@ -7,38 +7,86 @@
 import {
   // @ts-ignore: EuiBreadcrumbs has no exported member
   EuiBreadcrumbs,
-  EuiPanel,
-  EuiTitle,
+  EuiFlexGroup,
 } from '@elastic/eui';
+
 import React from 'react';
 import { pure } from 'recompose';
 import styled from 'styled-components';
+import chrome from 'ui/chrome';
+import { ESTermQuery } from '../../../common/typed_json';
+import { EmptyPage } from '../../components/empty_page';
 import { getHostsUrl, HostComponentProps } from '../../components/link_to/redirect_to_hosts';
+import { HostSummary } from '../../components/page/hosts/host_summary';
+import { manageQuery } from '../../components/page/manage_query';
+import { GlobalTime } from '../../containers/global_time';
+import { HostsQuery } from '../../containers/hosts';
+import { HostSummaryQuery } from '../../containers/hosts/host_summary.gql_query';
+import { indiciesExistOrDataTemporarilyUnavailable, WithSource } from '../../containers/source';
+import * as i18n from './translations';
+
+const basePath = chrome.getBasePath();
+
+const HostSummaryManage = manageQuery(HostSummary);
 
 export const HostDetails = pure<HostComponentProps>(({ match }) => (
-  <>
-    <HostBreadcrumbWrapper breadcrumbs={getBreadcrumbs(match.params.hostId!)} />
-    <EuiPanel>
-      <EuiTitle>
-        <h2>Host Details</h2>
-      </EuiTitle>
-      <div>
-        Match Params: <pre>{JSON.stringify(match, null, 2)}</pre>
-      </div>
-    </EuiPanel>
-  </>
+  <WithSource sourceId="default">
+    {({ auditbeatIndicesExist }) =>
+      indiciesExistOrDataTemporarilyUnavailable(auditbeatIndicesExist) ? (
+        <GlobalTime>
+          {({ poll, to, from, setQuery }) => (
+            <>
+              <HostBreadcrumbWrapper breadcrumbs={getBreadcrumbs(match.params.hostId!)} />
```

Comment:
If you don't do de-structuring, I don't think you need that bang on the end there do you? The less the better.

Reply:
I've updated to use destructuring as mentioned above, so this is cleaned up now.